platform/api/collection.go

```diff
@@ -30,6 +30,8 @@ const (
 	nextRelName    = "next"
 )
 
+const MAX_COMPLETION_RESULTS = 1000
```

Comment:
Post-GA we will be enforcing a maximum ?max query parameter value of 500 so this request will start to fail when that rule starts to be enforced.  We can either leave it as 1000 here and fix when the rule is enforced or we can set it to 500 and not worry about it.  What do you think is best here?